core-services/ci-secret-generator/_config.yaml

```diff
@@ -70,6 +70,10 @@
   attachments:
   - cmd: gcloud iam service-accounts keys create /dev/stdout --iam-account do-not-delete-ci-provisioner@openshift-gce-devel-ci.iam.gserviceaccount.com
     name: credentials.json
+- item_name: jenkins-ci-provisioner
+  attachments:
+  - cmd: gcloud iam service-accounts keys create /dev/stdout --iam-account do-not-delete-ci-xpn@openshift-gce-devel-ci.iam.gserviceaccount.com
+    name: credentials.json
```

Comment:
While this is "better", it isn't absolutely required; We can use `do-not-delete-ci-provisioner` for the installer to deploy the cluster while using `do-not-delete-ci-xpn` to attach to the gcp instances for "day 2" operations.

Reply:
I am a bit fuzzy on how the XPN secret gets passed in.  Looks like the XPN jobs get the vanilla `gcp` cluster profile, and the XPN secrets are not mentioned in `ci-operator/SECRETS.md`.  Can you explain where the key is currently stored in BitWarden and how it flows to the workflow steps that need it?

Reply:
@jstuever : bump :).  Can you clarify the current secret flow^?

/assign @jstuever

Reply:
It doesn't. The XPN installer currently uses do-not-delete-ci-provisioner for provisioning the initial just like the rest of CI. The do-not-delete-ci-xpn attached to bootstrap, controlPlane, and compute instances where they can then use it to do post-provisioning tasks such as creating ingress lbs.

Reply:
How does the cluster access the creds?  Does the ci-operator pod mount a secret with a token, or...?

Reply:
XPN is only available in UPI. The service accounts are attached to the instances in the UPI template[1], which gives the instance the access granted to the service accounts.

[1] https://github.com/openshift/installer/blob/9be316bc89f560653e3ad07cde7a716bf938cf6d/upi/gcp/05_control_plane.py#L26